frontend/src/components/Traveler/TravelerOverview.tsx

```diff
@@ -1,26 +1,27 @@
-import {useEffect} from "react";
-import {TravelerModel} from "./TravelerModel/TravelerModel";
+import { TravelerModel } from "./TravelerModel/TravelerModel";
 import TravelerCard from "./TravelerCard/TravelerCard";
 
 type TravelerOverviewProps = {
-    fetchAllTravelers: () => void,
-    travelers: TravelerModel[]
-}
-export default function TravelerOverview({fetchAllTravelers, travelers}: TravelerOverviewProps) {
-
-    useEffect(() => {
-        fetchAllTravelers()
-    }, [fetchAllTravelers])
-
-
-    const travelerList = travelers.map(traveler => {
-        return <TravelerCard key={traveler.id} fetchAllTraveler={fetchAllTravelers} traveler={traveler}/>
-    })
+  fetchAllTravelers: () => void;
+  travelers: TravelerModel[];
+};
+export default function TravelerOverview({
+  fetchAllTravelers,
+  travelers,
+}: TravelerOverviewProps) {
+  const travelerList = travelers.map((traveler) => {
     return (
-        <ul>
-            <h3>List of Persons:</h3>
-            {travelerList}
-        </ul>
-
+      <TravelerCard
+        key={traveler.id}
+        fetchAllTraveler={fetchAllTravelers}
+        traveler={traveler}
+      />
     );
+  });
+  return (
+    <ul>
+      <h3>List of Persons:</h3>
+      {travelerList}
```

Comment:
in my opinion it's more readable to write the .map right here, but it's just favor